3-enrich/spark-enrich/README.md

```diff
@@ -1,14 +1,14 @@
-# Scala Hadoop Enrich
+# Spark Enrich
 
 ## Introduction
 
-This is the Snowplow Enrichment process implemented for Hadoop using [Scalding][scalding]. The Hadoop Enrichment parses raw Snowplow event logs, extracts the Snowplow events, enriches them (e.g. with geo-location information) and then writes them out to Snowplow-format event files.
+This is the Snowplow Enrichment process implemented using [Spark] [spark]. The Hadoop Enrichment parses raw Snowplow event logs, extracts the Snowplow events, enriches them (e.g. with geo-location information) and then writes them out to Snowplow-format event files.
```

Comment:
Also few notes regarding READMEs:

* Scala Hadoop Enrich should be replaced with Spark Enrich in 3-enrich/README.md
* Scala Hadoop Shred should be removed from 3-enrich/README.md
* RDB Shredder should be added to 4-storage/README.md
* In SCE README we need to mention that it's used by Spark job and legacy Hadoop.